server/pages/index.js

```diff
@@ -120,7 +121,7 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		catchJsErrors: '/calypso/catch-js-errors-' + 'v2' + '.min.js',
+		catchJsErrors: '/calypso/catch-js-errors.js'
```

Comment:
I piped it through the regular bundler, as we do other assets, now we will be good :)